### ISSUE TITLE
Start section: SQS

### ISSUE THREAD

**Issue by rjhintz** [open-guides/og-aws#305]
Basic coverage of SQS, even just a placeholder.

[Quip collaborative doc - work in progress](https://og-aws.quip.com/94opAa8GnNbJ)

-------
[AWS SQS doc](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/Welcome.html)

[AWS SQS forum](https://forums.aws.amazon.com/forum.jspa?forumID=12)

See [SNS or SQS?](https://www.reddit.com/r/aws/comments/5bw7gn/sns_or_sqs/) comment stream for some interesting ways to process SQS items only once, using DynamoDB and Elasticache

aphyr on queues, in general: [Oh no, queues](https://github.com/aphyr/distsys-class)

-----

* What is a queue service?
* How does SQS implement a queue service?
* When does one use a queue service like SQS?
* What are important limits associated with SQS?

**Comment by k3ypad** [comment]
I can possibly take this - have requested access to Quip doc

**Comment by jlevy** [comment]
We now have this started with https://github.com/open-guides/og-aws/pull/505 thanks to @lobster1234 

@rjhintz @k3ypad Any follow-on PRs or feedback? I'll close this for now as I can't access that quip doc, but feel free to open a new PR or issue with additional changes!

**Comment by rjhintz** [comment]
I don't have any additional feedback at this time.

Rich
+1.510.984.2571
